# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../openssl/no-passwd.1.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- openssl-1.1.1h/doc/man1/passwd.pod.vanilla	2020-10-16 12:28:54.750071121 +0200
+++ openssl-1.1.1h/doc/man1/passwd.pod	2020-10-16 12:29:38.078072008 +0200
@@ -2,8 +2,7 @@
 
 =head1 NAME
 
-openssl-passwd,
-passwd - compute password hashes
+openssl-passwd - compute password hashes
 
 =head1 SYNOPSIS
 
